src/models/TechRecordsDAO.ts

```diff
@@ -86,7 +90,7 @@ class TechRecordsDAO {
 
     console.log("Query Params for getBySearchTerm ", query);
     try {
-      return this.queryAllData(query);
+      return await this.queryAllData(query);
```

Comment:
```suggestion
      return this.queryAllData(query);
```
don't know why this has been done, shouldn't return await right?